### PR TITLE
Add Metabiome's welcoming

### PR DESCRIPTION
**Pull request by biocdgv**
Hi Guys!! 

I have taken some risks and have created a new look for Metabiome's main script. Do you think is too much? I like it but I wanna know if you both like it too!! It is just a proposal!! I have made it through here [doodles](http://patorjk.com/software/taag/#p=display&v=2&f=ANSI%20Shadow&t=METABIOME%0A).

The changes are as follows: 

- Add Metabiome's bold welcoming to Metabiome main script
- Add issues' recommendation at the end of Metabiome main script

:broccoli: :broccoli: :broccoli: :broccoli: :broccoli: :broccoli: :broccoli:

### CODE REVIEW

File: scripts/metabiome.sh [hunk 1]
```diff
@@ -13,6 +13,18 @@ function usage() {
 cat<<HELP_USAGE
 Usage: metabiome [Commands|Options]
```
Comment (Nesper94) [inline scripts/metabiome.sh:15]: Remove one blank line:
```suggestion
```

File: scripts/metabiome.sh [hunk 2]
```diff
@@ -13,6 +13,18 @@ function usage() {
...
+
+			███╗   ███╗███████╗████████╗ █████╗ ██████╗ ██╗ ██████╗ ███╗   ███╗███████╗
+			████╗ ████║██╔════╝╚══██╔══╝██╔══██╗██╔══██╗██║██╔═══██╗████╗ ████║██╔════╝
+			██╔████╔██║█████╗     ██║   ███████║██████╔╝██║██║   ██║██╔████╔██║█████╗
+			██║╚██╔╝██║██╔══╝     ██║   ██╔══██║██╔══██╗██║██║   ██║██║╚██╔╝██║██╔══╝
+			██║ ╚═╝ ██║███████╗   ██║   ██║  ██║██████╔╝██║╚██████╔╝██║ ╚═╝ ██║███████╗
+			╚═╝     ╚═╝╚══════╝   ╚═╝   ╚═╝  ╚═╝╚═════╝ ╚═╝ ╚═════╝ ╚═╝     ╚═╝╚══════╝
+
+				A flexible and modular pipeline for metagenomic analysis!!!
+
```
Comment (Nesper94) [inline scripts/metabiome.sh:27]: Tabs should be replaced with spaces. Line length should be of max 80 chars, as it does not look fine on terminals with default size:
Selection_001

File: scripts/metabiome.sh [hunk 3]
```diff
@@ -48,6 +60,9 @@ Usage: metabiome [Commands|Options]
     -h, --help    Show this help.
     -v, --version Show Metabiome's version.
 
+If you are having problems running Metabiome, please do not hesitate to contact us through
+github (https://github.com/Nesper94/Metabiome)
```
Comment (Nesper94) [inline scripts/metabiome.sh:64]: Same line length suggestion:
```suggestion
If you are having problems running Metabiome, please do not hesitate to
contact us through GitHub (https://github.com/Nesper94/Metabiome)
```

**Review by Nesper94 — CHANGES_REQUESTED** [review]
Hi! Looks very nice! :sauropod:  Here some suggestions:

**Review by Estefany-Lorenzana — DISMISSED** [review]
Hi guys! It looks really good! What do you think about add in help function some welcome like this but smaller and with the name of the tool to us?

File: scripts/metabiome.sh [hunk 4]
```diff
@@ -13,6 +13,17 @@ function usage() {
 cat<<HELP_USAGE
 Usage: metabiome [Commands|Options]
 
+				Welcome to:
```
[the remark this answers is not in the document]
Comment (biocdgv) [reply scripts/metabiome.sh:16]: Indeed juan!! i will do it asap.

**Comment by biocdgv** [comment]
> Hi guys! It looks really good! What do you think about add in help function some welcome like this but smaller and with the name of the tool to us?

Hellow Stefa!! Do you mean adding to, for example, bowtie2, the same welcome as Metabiome? Like the following but smaller

imagen

Also, I haven't been able to reduce the font size of the Metabiome's welcoming...dunno if it's possible...

:wrench:

**Review by Nesper94 — APPROVED** [review]
It is great! I will also think in a way to implement the @Estefany-Lorenzana's proposal.

**Comment by Estefany-Lorenzana** [comment]
> > Hi guys! It looks really good! What do you think about add in help function some welcome like this but smaller and with the name of the tool to us?
> 
> Hellow Stefa!! Do you mean adding to, for example, bowtie2, the same welcome as Metabiome? Like the following but smaller
> 
> imagen
> 
> Also, I haven't been able to reduce the font size of the Metabiome's welcoming...dunno if it's possible...
> 
> wrench

Hi! Yeah, that's what I think!

**Review by Estefany-Lorenzana — APPROVED** [review]

**Pull request merged**